antora/docs/modules/ROOT/pages/release_policy.adoc

```diff
@@ -863,12 +865,25 @@ Confirm the CycloneDX SBOM contains only allowed packages. By default all packag
 * Code: `sbom_cyclonedx.allowed`
 * https://github.com/enterprise-contract/ec-policies/blob/{page-origin-refhash}/policy/release/sbom_cyclonedx.rego#L53[Source, window="_blank"]
 
+[#sbom_cyclonedx__disallowed_package_attributes]
+=== link:#sbom_cyclonedx__disallowed_package_attributes[Disallowed package attributes]
+
+Confirm the CycloneDX SBOM contains only packages without disallowed attributes. By default all attributes are allowed. Use the "dissalowed_attributes" rule data key to provide a list of key-value pairs that forbid the use of an attribute set to the given value.
+
+*Solution*: Update the image to not use a disallowed package attributes.
+
+* Rule type: [rule-type-indicator failure]#FAILURE#
+* FAILURE message: `Package %s has the attribute %q set%s`
+* Code: `sbom_cyclonedx.disallowed_package_attributes`
+* Effective from: `2024-07-31T00:00:00Z`
+* https://github.com/enterprise-contract/ec-policies/blob/{page-origin-refhash}/policy/release/sbom_cyclonedx.rego#L93[Source, window="_blank"]
+
 [#sbom_cyclonedx__disallowed_packages_provided]
 === link:#sbom_cyclonedx__disallowed_packages_provided[Disallowed packages list is provided]
 
-Confirm the `disallowed_packages` rule data was provided, since it is required by the policy rules in this package.
+Confirm the `disallowed_packages` and `dissalowed_attributes` rule data were provided, since they are required by the policy rules in this package.
```

Comment:
```suggestion
Confirm the `disallowed_packages` and `disallowed_attributes` rule data were provided, since they are required by the policy rules in this package.
```